docs/docs/examples/src/layouts/inline/backgroundImage.js

```diff
@@ -0,0 +1,13 @@
+var module = new pathfora.Subscription({
+  id: 'message-inline',
+  layout: 'inline',
+  position: '.inline-module',
+  headline: 'Welcome!',
+  msg: 'Please sign up for our newsletter for more updates.',
+  variant: 4,
```

Comment:
Might be something we need to discuss in person, but what was the deciding factor on making this a separate variant? Given the existing patterns with variants, it does sort of make sense. But in general I think we want to avoid using variants in the future as they are kind of a bad API design, and I think we'd like to support background images on other variant modals such as content recommendations.